pkg/netpol/eval/internal/k8s/adminnetpol.go

```diff
@@ -319,3 +439,39 @@ func (anp *AdminNetworkPolicy) HasValidPriority() bool {
 	// current implementation satisfies k8s requirement
 	return anp.Spec.Priority >= minANPPriority && anp.Spec.Priority <= maxANPPriority
 }
+
+// CheckEgressConnAllowed checks if the input conn is allowed/passed/denied or not captured on egress
+func (anp *AdminNetworkPolicy) CheckEgressConnAllowed(dst Peer, protocol, port string) (res int, captured bool, err error) {
+	for _, rule := range anp.Spec.Egress {
+		rulePeers := rule.To
+		rulePorts := rule.Ports
+		ruleRes, captured, err := checkIfEgressRuleContainsConn(rulePeers, rulePorts, dst, string(rule.Action), protocol, port, false)
+		if err != nil {
+			return 0, false, anp.anpRuleErr(rule.Name, err.Error())
+		}
+		if !captured {
+			continue
+		}
+		return ruleRes, captured, nil
+	}
+	// getting here means the protocol+port was not captured
+	return Pass, false, nil
```

Comment:
confusing const action `Pass` for the case of not captured

Reply:
changed